packages/amplify-e2e-tests/schemas/iterative-push/add-sort-to-primary-key/final-schema.graphql

```diff
@@ -0,0 +1,5 @@
+type Blog @model {
+  id: ID! @primaryKey(sortKeyFields: ["name"]
+  name: String!
+  posts: [Post] @connection(keyName: "byBlog1", fields: ["id"])
```

Comment:
Is there any reason you're using V1 (`@connection`) and V2 (`@primaryKey`) directives on the same schema? Can we make it fully V2?

Reply:
woops... bad copy, fixed